mne/io/edf/edf.py

```diff
@@ -384,6 +388,27 @@ def _get_info(fname, stim_channel, eog, misc, exclude, preload):
     chs = list()
     pick_mask = np.ones(len(ch_names))
 
+    # common channel type names mapped to internal ch types
+    ch_type_mapping = {
+        'EEG': FIFF.FIFFV_EEG_CH,
+        'SEEG': FIFF.FIFFV_SEEG_CH,
+        'ECOG': FIFF.FIFFV_ECOG_CH,
+        'DBS': FIFF.FIFFV_DBS_CH,
+        'EOG': FIFF.FIFFV_EOG_CH,
+        'ECG': FIFF.FIFFV_ECG_CH,
+        'EMG': FIFF.FIFFV_EMG_CH,
+        'BIO': FIFF.FIFFV_BIO_CH,
+        'RESP': FIFF.FIFFV_RESP_CH,
+        'MISC': FIFF.FIFFV_MISC_CH,
+        'SAO2': FIFF.FIFFV_BIO_CH,
+    }
+    bad_map = dict()
+
+    # montage is not able to be stored in EDF, so
+    # default to unknown locations for the channels
+    nan_arr = np.zeros(12)
```

Comment:
Why `12`?

Also, if you want an array of all nans consider using `np.full`.

Reply:
mne uses 12 values for channel locations as it is what you need for MEG. You need location and rotation matrix.

Reply:
Yes, I realized that, I thought a comment would have been nice. But it's all good, this is really minor.